torchvision/csrc/io/image/cpu/encode_jpeg.cpp

```diff
@@ -15,11 +15,7 @@ torch::Tensor encode_jpeg(const torch::Tensor& data, int64_t quality) {
 #else
 // For libjpeg version <= 9b, the out_size parameter in jpeg_mem_dest() is
 // defined as unsigned long, whereas in later version, it is defined as size_t.
-// For windows backward compatibility, we define JpegSizeType as different types
-// according to the libjpeg version used, in order to prevent compilation
-// errors.
-#if defined(_WIN32) || !defined(JPEG_LIB_VERSION_MAJOR) || \
```

Comment:
No more Windows checks? Ok with me, but IIUC there used to be discrepancies between Windows / Linux libjpeg that we might need to take into account?

Reply:
The situation [seems](https://app.circleci.com/pipelines/github/pytorch/vision/10521/workflows/d7d80f03-bef3-4e0f-a587-bc274097ed25/jobs/798797) to have been resolved in later versions of Windows (9d>=) hence blunt windows filter needs to go. I'll pin different versions of JPEG in a bit to see what exactly is the right condition we should use here.